motion_odometry:
  x_forward_scaling: {
    type: double,
    default_value: 1.25,
    description: "Scaling factor in forward x direction",
    validation: {
      bounds<>: [0.5, 1.5]
    }
  }

  x_offset: {
    type: double,
    default_value: 0.0,
    description: "Offset in x direction",
    validation: {
      bounds<>: [-1, 1]
    }
  }

  x_backward_scaling: {
    type: double,
    default_value: 0.95,
    description: "Scaling factor in backward x direction in m/second",
    validation: {
      bounds<>: [0.5, 1.5]
    }
  }

  y_scaling: {
    type: double,
    default_value: 1.0,
    description: "Scaling factor in y direction",
    validation: {
      bounds<>: [0.5, 1.5]
    }
  }
  y_offset: {
    type: double,
    default_value: 0.0,
    description: "Offset in y direction in m/second",
    validation: {
      bounds<>: [-1, 1]
    }
  }

  yaw_scaling: {
    type: double,
    default_value: 0.6,
    description: "Scaling factor for rotation in yaw direction",
    validation: {
      bounds<>: [0.2, 2.5]
    }
  }

  yaw_offset: {
    type: double,
    default_value: 0.0,
    description: "Offset in yaw direction in rad/second",
    validation: {
      bounds<>: [-3.14, 3.14]
    }
  }

  publish_walk_odom_tf: {
    type: bool,
    default_value: false,
    description:  "Should odom tf be published"
  }
